### DISCUSSION TITLE
MacOS ARM support

### DISCUSSION THREAD

**Discussion by pol2095** [airsdk/Adobe-Runtime-Support#1311] [Q&A] [answered]
Hello,

Adobe air support MacOS ARM natively or use Rosetta 2 ?
If yes, I don't see MacOS-ARM when I compile an ANE, only MacOS-x86, MacOS-x86-64 (is-it an universal binary) ?

Thanks

**Comment by ajwfrost** [comment]
Hi

If you're building an app using the AIR SDK and captive runtime, then we do support MacOS with ARM/Apple silicon. You'd need to have the ANEs set up as if for MacOS-x86-64 but with universal binaries containing x86_64 and arm64, for them to be loaded properly at runtime.

We debated having an option for MacOS-arm64 or MacOS-universal and may go that way but it's a little ambiguous what that might mean...

FYI the shared AIR runtime is still only x86_64 so if you have a .air file installed that way, it will just be using Rosetta2.

thanks

**Comment by pol2095** [comment, marked as answer]
Thanks
